content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -146,7 +146,7 @@ that container.
 
 ## Pod conditions
 
-A Pod has a PodStatus, which has an array of
+Apart from the single general value in Pod Phase field, the PodStatus includes an array of
```

Comment:
Hello @rozcietrzewiacz .
I am not sure if this is clear. Are you referring to the `phase` field (string) in PodStatus?
I think this is different from the strings describing the Pod conditions. Do you agree?
From the API reference:
```
The phase of a Pod is a simple, high-level summary of where the Pod is in its lifecycle. The conditions array, the reason and message fields, and the individual container status arrays contain more detail about the pod's status.
```
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#podstatus-v1-core

Reply:
Yes, I'm referring to `pod.status.phase` as mentioned in the PR description. I do understand it is different from `pod.status.conditions[*].type` - but when I first read this doc, I was a little confused, hence this PR.

Yes, the possible strings are different, but the slight confusion was around why are there two String fields describing the possible Pod states. Of course anyone can go and read the API reference, but, as I understand, the docs are supposed to make the architecture more digestible.

Maybe I didn't phrase it clearly enough. What I wanted to be clear at this place in the document, was that the `pod.status.phase` is a general **single** value describing the **current** Pod state, whereas `pod.status.conditions` is an array of objects identifying more detailed checkpoints that can be passed in the pod's lifecycle.

Do you see what I mean? How would you write it?

Reply:
Hi @rozcietrzewiacz . Without changing the text too much, how about something like:
A Pod has a PodStatus object which has two fields related to the Pod lifecycle: `conditions`
and `phase`. The `PodStatus.phase` field contains a high-level description of the current state of the Pod. 
The [`PodStatus.conditions`](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podcondition-v1-core) field provides a list of detailed information about the state through which the Pod has or has not passed.